rts/motoko-rts/src/gc/mark_compact/bitmap.rs

```diff
@@ -133,7 +133,8 @@ pub unsafe fn iter_bits() -> BitmapIter {
     let current_word = if blob_len_64bit_words == 0 {
```

Comment:
The fact that 'current_word' is a `u64` field in `BitMapIter` doesn't matter, right? Because we never write a BitMapIter to a (only 32-bit aligned) heap cell, and Rust will align it all correctly for its own use.